vagrant/ansible/roles/samba-glusterfs.setup/templates/smb.conf.j2

```diff
@@ -17,3 +17,12 @@ printcap name = /dev/null
 disable spoolss = yes
 printing = bsd
 show add printer wizard = no
+
+[gluster-{{ gluster_cluster_volume }}]
```

Comment:
Would love to see the whole share as a variable.

We would also have to change the location using hard coded share name gluster-vol which I believe is simply the exported sharenames in test-info.yml. 
vagrant/ansible/roles/client.prep/tasks/main.yml

and to create a series of test mounts in 
vagrant/ansible/roles/client.prep/tasks/mkdir_usermounts.yml
Although I would like to remove this all together as the this is better handled in the test runner instead.

Reply:
I've posted another PR #87  which affects the request I made above. 
It removes the code mentioned in 2) above.